### ISSUE TITLE
Disable carousel autoadvance

### ISSUE THREAD

**Issue by PeterBreen** [DemocracyLab/CivicTechExchange#957]
Per the accessibility team, we should disable the carousel automatic advance, meaning the only way to advance a slide is manually.

Our version of react-bootstrap (1.6, using Bootstrap 4.x) may not support `interval={null}` to disable autoadvance; that seems to be a react-bootstrap 2.x/Bootstrap 5.x function. However, if we can use null, let's do so.

To close this item, two things are required:
- remove the ability to set a custom interval via props for the carousel components
- remove any props.intervals from instances of those components.
- test that carousels effectively do not auto advance, and only change slides on user action (clicking the arrow)

If interval={null} does not work, set interval={315569520000}. When I discussed this with the accessibility team, they allowed as how users are _probably_ not going to wait ten years for a slide to advance. ;)

We have two Carousel components which need work: TestimonialCarousel and BlogCarousel.

## code locations
First, remove the logic allowing for custom intervals and simply have it set it to `interval={null}`, then remove interval from our type statement; we won't accept custom values via props anymore.

TestimonialCarousel:

 - https://github.com/DemocracyLab/CivicTechExchange/blob/1deba2dff9a48802e763b7e415af571207dfc37b/common/components/common/carousel/TestimonialCarousel.jsx#L37
 - https://github.com/DemocracyLab/CivicTechExchange/blob/1deba2dff9a48802e763b7e415af571207dfc37b/common/components/common/carousel/TestimonialCarousel.jsx#L8-L9

BlogCarousel:
 - https://github.com/DemocracyLab/CivicTechExchange/blob/1deba2dff9a48802e763b7e415af571207dfc37b/common/components/common/carousel/BlogCarousel.jsx#L36
 -  https://github.com/DemocracyLab/CivicTechExchange/blob/1deba2dff9a48802e763b7e415af571207dfc37b/common/components/common/carousel/BlogCarousel.jsx#L8-L9

Second, remove any use of props.interval:

  - https://github.com/DemocracyLab/CivicTechExchange/blob/2c5651d6b3a85f750e06a903a0c47d80a3c9ebdf/common/components/controllers/CorporateHackathonController.jsx#L264-L266
  - https://github.com/DemocracyLab/CivicTechExchange/blob/2c5651d6b3a85f750e06a903a0c47d80a3c9ebdf/common/components/controllers/CorporateHackathonController.jsx#L410-L413
 - https://github.com/DemocracyLab/CivicTechExchange/blob/3b4822018c1cdbc59fac7c46d4337ac32e6117b0/common/components/controllers/LandingController.jsx#L366
 
Third and finally, remove props.interval from this component which doesn't use it and really doesn't need it declared. This is just a cleanup task, but it's worth doing: 
 - https://github.com/DemocracyLab/CivicTechExchange/blob/2c5651d6b3a85f750e06a903a0c47d80a3c9ebdf/common/components/componentsBySection/Landing/LatestBlogPosts.jsx#L6-L9 

In short, set Carousels to have no autoadvance interval and remove support for custom autoadvance intervals. 

## reference documentation
https://react-bootstrap-v4.netlify.app/components/carousel/
